nodejs/week1/MealWebsite/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "mealwebsite",
+  "version": "1.0.0",
+  "description": "creating mealshare website",
+  "main": "src/backend/app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
You forgot to add the [test script](https://raw.githubusercontent.com/HackYourFuture-CPH/node.js/master/week1/homework/test/routes.spec.js). 🙂  Add it in the root of the MealWebsite folder, and just make sure that the `require()` paths in it are correct for all files that are required. Then replace this line with

```
     "test": "jest --runInBand" ,
```
and install jest and supertest
```
npm i jest supertest --save-dev
```
and do `npm run test`.